### PR TITLE
Add experimental-features.md

### PR DESCRIPTION
**Pull request by scsldb**
### What is changed, added or deleted? (Required)
Add experimental-features.md
Move performance file to 'About TiDB'

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: experimental-features.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Experimental Features
+summary: 了解 TiDB 4.0 版本的 Experimental Features。
+category: introduction
```
Comment (TomShawn) [inline experimental-features.md:4]:
```suggestion
```

File: experimental-features.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Experimental Features
+
+## 核心特性
```
Comment (TomShawn) [inline experimental-features.md:9]: 是否可以考虑去掉这个标题？

File: TOC.md [hunk 3]
```diff
@@ -8,7 +8,13 @@
 + 关于 TiDB
   + [TiDB 简介](/overview.md)
   + [What's New in TiDB 4.0](/whats-new-in-tidb-4.0.md)
+  + [Experimental Features in TiDB 4.0](/experimental-features.md)
```
Comment (TomShawn) [inline TOC.md:11]:
```suggestion
  + [TiDB 4.0 实验特性](/experimental-features.md)
```

File: experimental-features-4.0.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## 调度功能
+
++ Cascading Placement Rules 是一套副本规则系统，用于指导 PD 针对不同类型的数据生成对应的调度。通过组合不同的调度规则，用户可以精细地控制任何一段连续数据的副本数量、存放位置、主机类型、是否参与 Raft 投票、是否可以担任 Raft leader 等属性。详情参阅：[Cascading Placement Rules](/configure-placement-rules.md)。
++ 弹性调度功能，结合 Kubernetes，可根据实时负载状态，动态扩缩节点，能够有效地缓解业务高峰的压力并且节约不必要的成本开销。
+
+## SQL 功能
+
+- 支持表达式索引 (Expression Index) 功能，表达式索引也叫函数索引，在创建索引时索引的字段不一定要是一个具体的列，而可以由一个或者多个列计算出来的表达式。对于快速访问那些基于计算结果的表非常有用。详情参阅：[表达式索引](/sql-statements/sql-statement-create-index.md)。
+- 支持 AutoRandom Key 作为 TiDB 在列属性上的扩展语法，AutoRandom 被设计用于解决自增主键列的写热点问题，为使用自增主键列的用户提供最低成本的 MySQL 迁移方案。详情参阅：[AutoRandom Key](/auto-random.md)。
```
Comment (kissmydb) [inline experimental-features-4.0.md:18]: 确认过，AutoRandom 已经在 4.0.3 GA 了

File: whats-new-in-tidb-4.0.md [hunk 5]
```diff
@@ -55,8 +53,6 @@ TiUP 是 4.0 版本中新推出的包管理器的工具，主要用于管理 TiD
 - 新增查询数据时将 Join、Sort 中间结果写入本地磁盘，防止查询语句占用内存过多导致系统 OOM 的问题，提升系统的稳定性。
 - 优化 `EXPLAIN` 和 `EXPLAIN ANALYZE` 的输出结果，显示更多的信息，提升排查问题的效率。详情参阅：[Explain Analyze](/sql-statements/sql-statement-explain-analyze.md)，[Explain](/sql-statements/sql-statement-explain.md)。
 - 支持 Index Merge 功能，Index Merge 是一种新的表访问方式，当查询只涉及到单张表时，优化器会自动根据查询条件读取多个索引数据并对结果求并集，提升查询单张表时的性能。详情参阅：[Index Merge](/query-execution-plan.md#indexmerge-示例)。
-- 支持表达式索引 (Expression Index) 功能，表达式索引也叫函数索引，在创建索引时索引的字段不一定要是一个具体的列，而可以由一个或者多个列计算出来的表达式。对于快速访问那些基于计算结果的表非常有用（实验特性）。详情参阅：[表达式索引](/sql-statements/sql-statement-create-index.md)。
-- 支持 AutoRandom Key（实验特性）作为 TiDB 在列属性上的扩展语法，AutoRandom 被设计用于解决自增主键列的写热点问题，为使用自增主键列的用户提供最低成本的 MySQL 迁移方案。详情参阅：[AutoRandom Key](/auto-random.md)。
```
Comment (kissmydb) [inline whats-new-in-tidb-4.0.md:59]: 同上，AutoRandom 已经 GA，这里应该加回来

**Pull request merged**